app.drey.Warp.yaml

```diff
@@ -0,0 +1,45 @@
+---
+app-id: app.drey.Warp
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: warp
+finish-args:
+  # X11 and Wayland
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  # OpenGL rendering
+  - --device=dri
+  # Share files over the network
+  - --share=network
+  # Save files in the Downloads directory
+  - --filesystem=xdg-download
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  build-args:
+    - --share=network
+  test-args:
+    - --socket=x11
+    - --share=network
```

Comment:
```suggestion
```

I think my suggestion was unfinished. I really mean to remove the whole block.

Reply:
Ah sorry, I misunderstood that.